src/interface/src/app/treatments/treatments.state.ts

```diff
@@ -46,6 +46,8 @@ export class TreatmentsState {
   private _projectAreaId$ = new BehaviorSubject<number | undefined>(undefined);
   public projectAreaId$ = this._projectAreaId$.asObservable();
 
+  public planId$ = new BehaviorSubject<any>(null);
```

Comment:
types here please :)

Reply:
Ohh sorry! You will see that in my next commit